README.rst

```diff
@@ -68,6 +68,12 @@ install the dictionary::
 
 .. __: http://developer.postgresql.org/pgdocs/postgres/extend-extensions.html
 
+Using the PGXN client
+---------------------
+
+You can use the PGXN Client to download, build, and install 'italian_fts', e.g.::
```

Comment:
This doesn't need "build" I guess.

I checked the command out. It works but only if you are a superuser. I wonder if we shouldn't suggest:

```
pgxn install --sudo -- italian_fts
```

And to add
```
CREATE EXTENSION italian_fts;
```
to run in the database where it has to be used.

Reply:
> `pgxn install --sudo -- italian_fts`

or `sudo pgxn install italian_fts`